src/pages/commits/ActiveCommits/BuildVariantCard.tsx

```diff
@@ -67,34 +89,52 @@ const RenderGroupedIcons = ({ tasks }) => {
   );
   return (
     <>
-      {otherTasks.map(({ count, umbrellaStatus }) => (
-        <GroupedTaskStatusBadgeWrapper>
-          <GroupedTaskStatusBadge
-            status={umbrellaStatus}
-            key={`${umbrellaStatus}_groupedBadge`}
-            count={count}
-          />
-        </GroupedTaskStatusBadgeWrapper>
-      ))}
+      {otherTasks.map(({ count, umbrellaStatus }) => {
+        const onClick = () =>
+          push(
+            getVersionRoute(versionId, {
+              statuses: mapUmbrellaStatusToQueryParam[umbrellaStatus],
+              variant: buildVariantDisplayName,
+            })
+          );
+        return (
+          <GroupedTaskStatusBadgeWrapper>
+            <GroupedTaskStatusBadge
+              status={umbrellaStatus}
+              key={`${umbrellaStatus}_groupedBadge`}
```

Comment:
We should also include versionId and buildVariant in this key to ensure uniqueness. Since there is a chance there can be multiple badges with the same umbrella status